zipf/bin/plotcounts.py

```diff
@@ -76,15 +76,18 @@ def main(args):
     ax = df.plot.scatter(x='word_frequency', y='rank', loglog=True,
                          figsize=[12, 6], grid=True, xlim=args.xlim)
 
-    alpha = get_power_law_params(df['word_frequency'].to_numpy())
+    alpha, beta = get_power_law_params(df['word_frequency'].to_numpy())
```

Comment:
```suggestion
    alpha = get_power_law_params(df['word_frequency'].to_numpy())
```
In older versions of `plotcounts.py`, `get_power_law_params` returned both `alpha` and `beta`, but now it only returns `alpha`.